doi_to_org.py

```diff
@@ -204,6 +209,20 @@ def tempfile_from_path(cls, path):
         shutil.copyfile(path, f.name)
         return f
 
+    @classmethod
+    def tempfile_from_key(cls, path, key):
+        if not os.path.isdir(path):
+            raise FileError('Not a valid path.')
+
+        for root, dirs, files in os.walk(path):
+            for file in files:
+                if file.endswith(".pdf") and file == "%s.pdf" % (key):
+                    f = tempfile.NamedTemporaryFile()
+                    shutil.copyfile(os.path.join(root, file), f.name)
+                    return f
+
+        sys.exit("Couldn't find file '%s.pdf' in '%s'" % (key, path))
```

Comment:
I think it would be better to raise an exception here, like `FileNotFound`, and if we catch it later then calling `sys.exit`.

Reply:
We could easily allow another extension (or no extension at all) here. We just need to replace the conditoin from line 219 by a `os.path.splitext(file)[0] == key`.

Reply:
Maybe rename `file`, it is already a class in Python 2.